### DISCUSSION TITLE
Simple child machine doesn't receive events

### DISCUSSION THREAD

**Discussion by volisoft** [statelyai/xstate#3469] [Q&A]
Playing with a simple machine setup, found out that child machine doesn't receive events, e.g. `dblclick` when parent machine is in `text_edit` state. Tested it both in inspector and in code with no luck. Could someone please explain what's wrong with the setup?
```
import { createMachine, Machine, assign } from "xstate";

const textMachine = Machine({
  id: "text",
  initial: "disabled",
  context: { id: null, text: null },
  states: {
    disabled: { on: { dblclick: { target: "editing" } } },
    editing: { on: { esc: { target: "commit" } } },
    commit: { type: "final" },
  },
});

const fetchMachine = Machine({
  id: "tbl",
  initial: "initial",
  context: {
    retries: 0,
  },
  states: {
    initial: {
      on: {
        table_edit: { target: "initial" },
        text_edit: { target: "text_edit" },
      },
    },
    text_edit: {
      invoke: {
        id: "text",
        src: textMachine,
      },
      onDone: {
        target: "initial",
      },
    },
  },
});
```

**Comment by Andarist** [comment]
Please always try to share a repro case in a runnable form - either by providing a git repository to clone or a codesandbox. OSS maintainers usually can't afford the time to set up the repro, even if exact steps are given.

From the given piece of code it's unclear to me how do you try to send events to this child machine.
